cmd/podman/varlink/io.projectatomic.podman.varlink

```diff
@@ -609,13 +642,126 @@ method ExportImage(name: string, destination: string, compress: bool, tags: []st
 # ~~~
 method PullImage(name: string) -> (id: string)
 
+# CreatePod creates a new empty pod.  It takes name and cgroup_parent args as strings, and a labels map
+# On success, the ID of the newly created pod will be returned.
+# #### Example
+# ~~~
+# $ varlink call unix:/run/podman/io.projectatomic.podman/io.projectatomic.podman.CreatePod '{"name": "test"}'
+# {
+#   "pod": "8759dafbc0a4dc3bcfb57eeb72e4331eb73c5cc09ab968e65ce45b9ad5c4b6bb"
+# }
+# ~~~
+method CreatePod() -> (notimplemented: NotImplemented)
+
+# ListPods returns a list of pods in no particular order.  They are
+# returned as an array of ListPodData structs.  See also [GetPod](#GetPod).
+method ListPod() -> (notimplemented: NotImplemented)
+
+# GetPod takes a name or ID of a pod and returns single [ListPodData](#ListPodData)
+# structure.  A [PodNotFound](#PodNotFound) error will be returned if the pod cannot be found.
+# See also [ListPods](ListPods).
+method GetPod() -> (notimplemented: NotImplemented)
+
+# InspectPod takes the ID or name of a pod and returns one [InspectPodData](#InspectPodData)
+# structure.  A [PodNotFound](#PodNotFound) error will be returned if the pod cannot be found.
+# See also [ListPods](ListPods).
+# Note: this structure is not implemented because podman pod inspect has not been implemented.
+method InspectPod() -> (notimplemented: NotImplemented)
```

Comment:
InspectImage() and InspectContainer() return JSON blobs.  InspectPod is going to be a struct?